spec/integ/megolm-integ.spec.ts

```diff
@@ -1463,3 +1348,142 @@ describe("megolm", () => {
         ]);
     });
 });
+
+/**
+ * Establish an Olm Session with the test user
+ *
+ * Waits for the test user to upload their keys, then sends a /sync response with a to-device message which will
+ * establish an Olm session.
+ */
+async function establishOlmSession(aliceTestClient: TestClient, bobOlmAccount: Olm.Account): Promise<Olm.Session> {
```

Comment:
I think when factoring this out into a function, it'd be better to give these arguments names that describe their actual roles, rather than "alice" and "bob".

Reply:
done in [2328d11](https://github.com/matrix-org/matrix-js-sdk/pull/2958/commits/2328d11eafc0987db96b12e20bc432d7279b39cd)